### ISSUE TITLE
Feature request: Consistent naming of SERVER_COUNTRIES

### ISSUE THREAD

**Issue by ezekieldas** [qdm12/gluetun#1944]
### What's the feature 🧐

This is a "nice to have", particularly in the area of automation of containers and interoperability with other applications. 

I'm exploring different service providers (`VPN_SERVICE_PROVIDER`) and straight away, between ivpn and Mullvad I've found the former requires 'United States' and the latter, 'USA' in a `SERVER_COUNTRIES` definition. 

At cursory glance, it appears our friends at ivpn ([ivpn API](https://api.ivpn.net/v4/servers.json)) are using Alpha-2 codes, as described here: 

https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes

I can research further if there are strong recommendations in terms of using this in software applications, databases, etc. 

Other thoughts on this:

-  I haven't looked but wouldn't be surprised to find libraries that translate these names (eg, 'United Kingdom', 'England' = GB)
-  Perhaps another chapter but city names as well? [IATA Codes](https://stackoverflow.com/questions/7066825/is-there-an-iso-standard-for-city-identification) perhaps?

**Comment by misaligar** [comment]
I am entering all different variations in the environments tab of the Portainer but a random server is selected every time I restart the gluetun container. Any thoughts? So far I've tried:

United States
united states
"United States"
'United States'
'united states'
"united states"
USA
'USA'
"USA"

**Comment by qdm12** [comment]
Good idea, it would be nice to have synonyms basically I would say, to keep retro-compatibility and let users a bit more freedom in their filter values, especially when it comes to countries. We can add them manually as they are requested I guess. Keeping this opened, but obviously not a priority right now 😄 Thanks!
